tests/e2e/default/enketo/submit-form.wdio-spec.js

```diff
@@ -118,7 +119,7 @@ describe('Submit Enketo form', () => {
     await commonElements.openFastActionReport(assessmentForm.internalId, false);
 
     // enter name
-    await (await genericForm.nameField()).setValue('Jones');
+    await commonEnketoPage.setInputValue('person.field.name', 'Jones');
```

Comment:
Seems odd. Is this label not translated in the config?

Reply:
It has that weird label because it is a custom form, it is not part of the config, and that is the label that the form has -> see [here](https://github.com/medic/cht-core/blob/8697-consolidate-enketo-common-selectors/tests/e2e/default/enketo/submit-form.wdio-spec.js#L31).

But you are right, it might be confusing to see that in the test, I will change it.